javaparser-core/src/main/java/com/github/javaparser/ast/validator/chunks/ModifierValidator.java

```diff
@@ -49,7 +49,7 @@ public void visit(ClassOrInterfaceDeclaration n, ProblemReporter reporter) {
 
     private void validateClassModifiers(ClassOrInterfaceDeclaration n, ProblemReporter reporter) {
         if (n.isTopLevelType()) {
-            validateModifiers(n, reporter, PUBLIC, ABSTRACT, FINAL, STRICTFP);
+            validateModifiers(n, reporter, PUBLIC, ABSTRACT, STATIC, FINAL, STRICTFP);
```

Comment:
I was hoping that we could change the stub file format to be the same as a Java file.  That is, instead of a stub file containing

```
public class Outer { ... }
public class Outer$Inner { ... }
```

the stub file would contain

```
public class Outer { ...
  public class Inner { ... }
}
```

Is that possible?